assignments/stretch-function-conversion.js

```diff
@@ -1,23 +1,32 @@
 // Take the commented ES5 syntax and convert it to ES6 arrow Syntax
```

Comment:
I'm thrilled you decided to do this stretch. I'm going to ask everyone to go back and do this because it's so important to understand and get use to doing/reading. 

fantastic.